source/docs/api-reference/wpilib-integration/motorcontroller-integration.rst

```diff
@@ -0,0 +1,79 @@
+MotorController Integration
+===========================
+
+Phoenix Pro motor classes such as ``TalonFX`` (`Java <https://api.ctr-electronics.com/phoenixpro/release/java/com/ctre/phoenixpro/hardware/TalonFX.html>`__, `C++ <https://api.ctr-electronics.com/phoenixpro/release/cpp/classctre_1_1phoenixpro_1_1hardware_1_1_talon_f_x.html>`__) implement the ``MotorController`` (`Java <https://github.wpilib.org/allwpilib/docs/release/java/edu/wpi/first/wpilibj/motorcontrol/MotorController.html>`__, `C++ <https://github.wpilib.org/allwpilib/docs/release/cpp/classfrc_1_1_motor_controller.html>`__) interface.
+
+Phoenix Pro implements all inherited functions:
+
+- ``set``
+- ``setVoltage``
+- ``get``
+- ``setInverted``
+- ``getInverted``
+- ``disable``
+- ``stopMotor``
+
+Due to inheriting the ``MotorController`` interface, this allows motors such as ``TalonFX`` to be included as part of the ``DifferentialDrive`` constructor.
+
+.. tab-set::
+
+   .. tab-item:: Java
+      :sync: Java
+
+      .. code-block:: java
+
+         // instantiate motor controllers
+         TalonFX m_motorLeft = new TalonFX(0);
+         TalonFX m_motorRight = new TalonFX(1);
+
+         // create differentialdrive object for robot control
+         DifferentialDrive m_diffDrive = new DifferentialDrive(m_motorLeft, m_motorRight);
+
+         // instantiate joystick
+         XboxController m_driverJoy = new XboxController(0);
+
+         public void teleopPeriodic() {
+            var forward = -m_driverJoy.getLeftY();
+            var rot = -m_driverJoy.getRightX();
+
+            m_diffDrive.arcadeDrive(forward, rot);
+         }
+
+   .. tab-item:: C++ (Source)
+      :sync: C++
+
+      .. code-block:: cpp
+
+         public void TeleopPeriodic() {
+            auto forward = -driverJoy.GetLeftY();
+            auto rot = -driverJoy.GetRightX();
+
+            diffDrive.ArcadeDrive(forward, rot);
+         }
+
+   .. tab-item:: C++ (Header)
+      :sync: C++ Header
+
+      .. code-block:: cpp
+
+         #include "frc/drive/DifferentialDrive.h"
+         #include "frc/XboxController.h"
```

Comment:
If we're going to have `#include`s, also add `#include "ctre/phoenixpro/TalonFX.hpp"`